package.json

```diff
@@ -13,22 +13,20 @@
         "start:server": "tinylicious"
     },
     "dependencies": {
-        "@fluid-experimental/data-objects": "^0.34.0-14942",
-        "@fluidframework/aqueduct": "^0.32.1",
-        "@fluidframework/get-tinylicious-container": "^0.32.1",
-        "@fluidframework/map": "^0.32.1",
+        "@fluid-experimental/data-objects": "^0.35.2",
+        "@fluid-experimental/fluid-static": "^0.35.2",
         "react": "^17.0.1",
         "react-dom": "^17.0.1",
         "vue": "^3.0.5"
     },
     "devDependencies": {
-        "@types/react": "^17.0.0",
-        "@types/react-dom": "^17.0.0",
+        "@types/react": "^17.0.2",
+        "@types/react-dom": "^17.0.1",
         "clean-webpack-plugin": "^3.0.0",
         "concurrently": "^5.3.0",
         "html-webpack-plugin": "^4.3.0",
-        "tinylicious": "0.4.11798",
-        "ts-loader": "^8.0.12",
+        "tinylicious": "0.4.17169",
```

Comment:
you can also add `npx tinylicious` to the readme steps and leave it out as a dependency